recipe/meta.yaml

```
@@ -1,8 +1,6 @@
 {% set name = "helm" %}
-{% set goname = "k8s.io/" + name %}
-
 {% set version = "3.16.2" %}
-{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+{% set pkg_src = os.sep.join("src", "k8s.io", name) %}
```

Ah sorry. Had missed the `list` before. Think this should fix the errors

```suggestion
{% set pkg_src = os.sep.join(["src", "k8s.io", name]) %}
```